src/main/webapp/dashboard.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Div of the About me tab.
+ */
+const aboutMeDiv = document.getElementById('about-me');
+
+/**
+ * Fake information for the hardcoded stage.
+ */
+const dummyName = 'Dior';
+const dummyLocation = 'Sunnyvale CA.';
+const dummyBio = 'The cutest pug in The tech industry. Lorem ipsum dolor sit ' +
+  'amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut' +
+  ' labore et dolore magna aliqua.';
+const dummyImg = 'images/dior.jpg';
+const dummyTags = ['Weekly', '30 mins', 'Kubernetes', 'Open Sesame',
+  'Documentation',
+];
+const dummyGitSrc = '#';
+
+/**
+ * Creates a card element containing information about a given user.
+ * @param {String} name
+ * @param {String} location
+ * @param {String} bio
+ * @param {String} imgSrc
+ * @param {String List} tags
+ * @return {HTMLElement} About me Card
+ */
+function createAboutMe(name, location, bio, imgSrc, tags, gitSrc) {
+  let AboutMeCard = document.createElement('div');
+  AboutMeCard.className = 'card card-holder col-12';
+
+  let AboutMeCardBody = document.createElement('div');
+  AboutMeCardBody.className = 'card-body';
+  AboutMeCard.append(AboutMeCardBody);
+
+  let editButton = document.createElement('a');
+  editButton.className = 'float-right bold';
+  editButton.innerText = 'Edit'
```

Comment:
Missing a ; (wow did the linter catch this too!?)

Reply:
Also, just a Q, but do you always want an about me card to always exist. jut with customized content? (if so, you could hard code the about me card in the page html, and only use a function like this to change the contents instead of creating a new elements. May make the function a bit shorter?)

Reply:
I'm just thinking about which of these divs could already exist, versus which you really need to create in this java script function.

Reply:
yeah that makes a lot of sense. I'll re-evaluate my divs. I'll probably create helper functions as well to improve readability

Reply:
```suggestion
  editButton.innerText = 'Edit';
```